x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/create_threat_signals.ts

```diff
@@ -66,47 +83,93 @@ export const createThreatSignals = async ({
     searchAfter: undefined,
     sortField: undefined,
     sortOrder: undefined,
+    logger,
+    buildRuleMessage,
+    perPage,
   });
 
-  while (threatList.hits.hits.length !== 0 && results.createdSignalsCount <= params.maxSignals) {
-    ({ threatList, results } = await createThreatSignal({
-      threatMapping,
-      query,
-      inputIndex,
-      type,
-      filters,
-      language,
-      savedId,
-      services,
+  while (threatList.hits.hits.length !== 0) {
+    const chunks = chunk(itemsPerSearch, threatList.hits.hits);
+    logger.debug(
+      buildRuleMessage(
+        `${chunks.length} concurrent indicator searches are starting. Each search can have up to ${itemsPerSearch} indicator items per search`
```

Comment:
Might be helpful to add how many that's out of? Like at first I saw `9 concurrent indicator searches...` and didn't know if I had set it wrong (it was set to 10). Super duper nit, just thought it might be helpful.

Reply:
I might drop this part of the message as later on it reports how many items within the search it is searching against which might be smaller than this number.

That part later is here:

```ts
logger.debug(
      buildRuleMessage(
        `${threatFilter.query.bool.should.length} indicator items are being checked for existence of matches`
      )
    ); 
```

So I think I am going to just cut off the extra bit here:

```ts
Each search can have up to ${itemsPerSearch} indicator items per search
```

To avoid confusing anyone who doesn't seen things add up from their logs.